src/components/OnboardingScreen.js

```diff
@@ -88,18 +96,18 @@ export function OnboardingScreen({
             )}
           >
             <div className="nfd-flex nfd-flex-col nfd-justify-start nfd-items-start nfd-gap-4">
-              <Title size="2">{title}</Title>
+              <Title size="2">{isMigrationCompleted ? Text.isMigrated.title : title}</Title>
               <div>
                 {comingSoon ? (
                   <Alert
                     variant="warning"
                     className="nfd-text-sm nfd-bg-transparent nfd-p-0 "
                   >
-                    <span className="nfd-text-red-700">{description}</span>
+                    <span className="nfd-text-red-700">{isMigrationCompleted ? Text.isMigrated.description : description}</span>
                   </Alert>
```

Comment:
Translations need to be added

Reply:
translations are added in the variable which is being used here